### PR TITLE
Add internationalization section to docs.

### LINKED ISSUE #963: Docs: Missing section about i18n
The topic is missingt. As good basis the [hands-on tutorial for Volto](https://training.plone.org/5/volto/i18n.html) can be re-used.

### PR DESCRIPTION
**Pull request by pgrunewald**
Fixes #963

### CODE REVIEW

**Comment by pgrunewald** [comment]
Done @tisto

**Comment by pgrunewald** [comment]
I tried retriggering all the checks, but it does not seem to be working. Also all conflicts should be resolved. This is not been noticed either.

Maybe someone can look into this or give me a hint? /cc @tisto @sneridagh

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -66,7 +67,6 @@ markdown_extensions:
   - pymdownx.magiclink
   - pymdownx.mark
   - pymdownx.details
-  - pymdownx.superfences
```
Comment (tisto) [inline docs/mkdocs.yml]: @pgrunewald any particular reason you removed pymdownx.superfences?

**Comment by tisto** [comment]
@pgrunewald I resolved the merge conflict.

File: docs/mkdocs.yml (hunk 1 above)
Comment (pgrunewald) [reply docs/mkdocs.yml]: @tisto: Because it was duplicated in the extension list. Unfortunately the diff view in GitHub didn't quite show this.

Comment (tisto) [reply docs/mkdocs.yml]: @pgrunewald ah, cool! Did not see this. Thanks for cleaning up! :)

**Comment by pgrunewald** [comment]
Also as side note: Force pushing a feature branch seems not so work well with the automatic checks. I tried to rebase instead of merging. Next time I directly stick to merge.

**Comment by tisto** [comment]
@pgrunewald merged. Thanks for your contribution!

**Pull request merged**